.github/workflows/test_prometheus_rules.yaml

```diff
@@ -0,0 +1,28 @@
+name: Test prometheus rules
```

Comment:
This needs to be called on `ci.yaml` no?:
```yaml
# ci.yaml
jobs:
  unit-tests:
  . . .
  alert-unit-tests:
    name: Alert tests
    uses: ./.github/workflows/test_prometheus_rules.yaml
```

Reply:
Sorry, for some reason I forgot to add this change.